src/services/catalog.service.ts

```diff
@@ -145,26 +145,56 @@ export class ImageItem implements IServiceItem {
   public description: string;
   public longDescription: string;
   public resource: any;
-  public isBuilder: boolean;
-  public isHidden: boolean;
+  private validSpecTagName: any;
```

Comment:
"Valid" seems like the wrong name. Non-builder tags are still valid. Maybe `builderSpecTagName`.